04_setup_ironic.sh

```diff
@@ -16,11 +17,18 @@ for IMAGE_VAR in IRONIC_IMAGE IRONIC_INSPECTOR_IMAGE IPA_DOWNLOADER_IMAGE COREOS
         # Clone to ~ if not there already
         [ -e "$REPOPATH" ] || git clone $IMAGE $REPOPATH
         cd $REPOPATH
-        export $IMAGE_VAR=localhost/${IMAGE##*/}:latest
+        export $IMAGE_VAR=${IMAGE##*/}:latest
         sudo podman build -t ${!IMAGE_VAR} .
         cd -
-    else
-        sudo podman pull "$IMAGE"
+        sudo podman push --tls-verify=false ${!IMAGE_VAR} localhost:5000/localimages/${!IMAGE_VAR}
+        export $IMAGE_VAR=192.168.111.1:5000/localimages/${!IMAGE_VAR}
+
+        clone_installer
+        cd /home/notstack/go/src/github.com/openshift/installer
+        mkdir -p data/data/bootstrap/baremetal/files/etc/containers
+        echo -e "[registries.insecure]\nregistries = ['192.168.111.1:5000']" > data/data/bootstrap/baremetal/files/etc/containers/registries.conf 
+        sed -i -e "s%$IMAGE_VAR=.*%$IMAGE_VAR=${!IMAGE_VAR}%g" data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template
```

Comment:
This won't work when https://github.com/openshift/installer/pull/2234 lands, could we change the logic to replace `$(image_for xyz)` instead?

Reply:
Yup, will update once its landed